### PR TITLE
Changes a lot of food names and descriptions. Removes alkysine in brainfood, substitutes with MBT. Will require a wiki update.

### PR DESCRIPTION
**Pull request by chaoko99**
SS13's dumb memes are garbage and I will fight you on this.

### CODE REVIEW

**Comment by printer16** [comment]
Disagreeing with most of the changes. It's a game.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -2477,19 +2424,6 @@
 	reagents.add_reagent("imidazoline", 3)
 	bitesize = 2
 
-/obj/item/weapon/reagent_containers/food/snacks/superbiteburger
```
Comment (printer16) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Why?

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -1070,6 +1070,7 @@
 	. = ..()
 	reagents.add_reagent("protein", 6)
 	reagents.add_reagent("alkysine", 6)
+	reagents.add_reagent("mindbreaker", 6) //Brain meat is rich in PRIONS.
```
Comment (printer16) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: This is more than a "meme removal change". Please log it.

File: code/modules/reagents/reagent_containers/food/snacks.dm (hunk 1 above)
Comment (chaoko99) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: That's the one burger that's larger than a person.

Comment (chaoko99) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: Also: meant to only remove the recipe. One moment.

**Comment by chaoko99** [comment]
Lost files in a revert. Don't care quite enough to fix that. Might do this again later.

**Pull request closed without merging**